nikola/plugins/misc/taxonomies_classifier.py

```diff
@@ -233,7 +233,7 @@ def _postprocess_path(self, path, lang, append_index='auto', dest_type='page', p
         # Forcing extension for Atom feeds and RSS feeds
         force_extension = None
         if dest_type == 'feed':
-            force_extension = '.atom'
+            force_extension = self.site.config['ATOM_EXTENSION']
         elif dest_type == 'rss':
             force_extension = '.xml'
```

Comment:
Shouldn’t this be be RSS_EXTENSION?  And what about the `['rss']` on line 243 (gh won’t let me comment there)?

Reply:
No, since `'feed'` is for Atom feeds (`'rss'` is for RSS feeds). But it looks like I forgot to change `.xml` two lines below...
And yes, the `['rss']` in line 243 also needs a change.

Reply:
Ah, ok :)